src/aplContainer/commands/createAplDocumentFromSample.ts

```diff
@@ -142,6 +142,7 @@ export class CreateAplDocumentFromSampleCommand extends AbstractCommand<void> {
                         id: k
                     } as SampleTemplateQuickPickItem)
             )
+            .filter((item) => !!item.label)
```

Comment:
Are these templates unusable ? Considering these templates are added specifically for APL 1.5 version, shouldn't  we provide these templates for skill devs rather than filtering them out ?

Reply:
Got to know that the new templates are test values and are not actually real templates for skill development. Resolving this conversation.